### ISSUE TITLE
Problems getting to site

### ISSUE THREAD

**Issue by JenM00re** [Swarthmore/a11y-doc-repo#64]
**Describe the bug**
Attempted to go to the site to test with ANDI.

**To Reproduce**
1. Go to https://swarthmore-a11ygator.vercel.app/
2. Blocked.
3. See error

**Expected behavior**
I should be able to login in order to accessibility test the site.

**Screenshots**

<img width="784" alt="'Your connection is not private' warning screen" src="https://github.com/user-attachments/assets/dac858ca-4f1b-400d-95be-36cdb2d96fd9" />

**Desktop (please complete the following information):**
 - OS: Mac 15.5
 - Browser Chrome 
 - Version 137.0.7151.120 (Official Build) (arm64)

**Comment by JenM00re** [comment]
Also experiencing on Firefox:
Desktop (please complete the following information):

OS: Mac 15.5
Browser Firefox
Version 128.6.0esr

**Comment by JenM00re** [comment]
Also experienced in Safari:
Desktop (please complete the following information):

OS: Mac 15.5
Browser Firefox
Version 128.6.0esr
